src/components/community/comments/components/comment-card/index.tsx

```diff
@@ -66,18 +66,25 @@ const CommentCard: FC<Props> = ({
   const hasReplies = (replies?.length ?? 0) > 0;
   const maxVisibleReplies = 3;
   const croppedReplies = showAllReplies
-    ? replies?.reverse()
-    : replies?.slice(0, maxVisibleReplies)?.reverse();
+    ? replies
+    : replies?.slice(-maxVisibleReplies);
 
   const sendPost = (content: string, mode: Mode) => {
     switch (mode) {
       case Mode.UPDATE:
-        updateComment({ id: entityId, post: { ...comment, content } });
+        updateComment({
+          id: entityId,
+          post: { ...comment, content },
+          page: comment.page ?? 1
```

Comment:
Hvorfor må page være knyttet til oppdatering av en kommentar?

Reply:
Dette er for å sørge for at cachen for den siden blir invalidert, og trigger ny fetch av data